packages/scheduler/jobs/refreshPerformanceData.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ ( -z "${QA_ENV}" || "${QA_ENV}" != true ) && "${NODE_ENV}" == "production" ]]; then
```

Comment:
Does the schedular package has these 2 env variables? And the value of env variables are always string you will probably need `"true"` instead of `true`